### ISSUE TITLE
Provide pre-built native binary

### ISSUE THREAD

**Issue by ByteHamster** [jherkenhoff/qalculate-android#11]
Currently, it takes a bit of fiddling to setup the build (installing cmake dependencies, etc). I think it would be easier to attract Android developers if the native library was a simple dependency from JitPack.

A few more advantages:

- Building (for the first time) gets faster
- Would make it easier to build the app on a CI server like GitHub Actions for automatic code review
- Would maybe even enable other apps using it as a library
- More consistent versioning (currently, the library repo is not "synchronized" with the app repo)

JitPack has a build limit of 15 minutes. That could get rather close. Maybe the build can be parallelized.

**Comment by ByteHamster** [comment]
Hmm, not sure if this is possible. They do not have `swig` installed on their servers. Installing with `apt` (with or without `sudo`) does not work. Then I tried building `swig` from source on JitPack but it internally needs `libpcre`, which is also not available. Finally, compiling `libpcre` from source ended with `No suitable c compiler found`, which is when I gave up. Maybe it can be built using GitHub actions.
